pkg/bootstrap/docker/up.go

```diff
@@ -47,8 +47,9 @@ const (
 	initialProjectDisplay = "My Project"
 	initialProjectDesc    = "Initial developer project"
 
-	defaultRedirectClient  = "openshift-web-console"
+	// developmentRedirectURI is a default redirect to enable out-of-the-box web-console development against a cluster up server
```

Comment:
Probably on the section that actually adds the redirect URIs (the task down below):

```
// adds a redirect URI to enable local console development.  This is not a security risk because XYZ
```

or something